libethereum/State.cpp

```diff
@@ -541,6 +541,7 @@ std::pair<ExecutionResult, TransactionReceipt> State::execute(EnvInfo const& _en
 	u256 const startGasUsed = _envInfo.gasUsed();
 	executeTransaction(e, _t, onOp);
 
+	res.changeLog = m_changeLog;
```

Comment:
yes, getter function would be nice. 
state.executeTransaction()
state.getChangelog()